simphony_mayavi/sources/cuds_source.py

```diff
@@ -89,6 +89,42 @@ def __vtk_cuds_changed(self, value):
 
     # Public method ########################################################
 
+    def __init__(self, cuds=None, point_scalars=None, point_vectors=None,
+                 cell_scalars=None, cell_vectors=None):
+        """Initialise the CUDSSource
+
+        Parameters
+        ----------
+        cuds : ABCParticles, ABCLattice, ABCMesh or H5Mesh, optional
+
+        point_scalars : str, optional
+            CUBA name of the data to be selected as point scalars.
+            Default is the first available point scalars.
+
+        point_vectors : str, optional
+            CUBA name of the data to be selected as point vectors.
+            Default is the first available point vectors.
+
+        cell_scalars : str, optional
+            CUBA name of the data to be selected as cell scalars.
+            Default is the first available cell scalars.
+
+        cell_scalars : str, optional
+            CUBA name of the data to be selected as cell scalars.
+            Default is the first available cell scalars.
+        """
+        # required by Traits
+        HasTraits.__init__(self)
```

Comment:
I was told that `HasTraits` is an old-style class that needs this weird treatment.